certifier_service/certlib/certlib_proofs.go

```diff
@@ -1153,6 +1153,54 @@ func VerifySevAttestation(serialized []byte, k *certprotos.KeyMessage) []byte {
 		  byte dev_public_key[PUBLIC_KEY_SIZE];
 		};
  */
+
+/*
+ * TODO: Notes taken during call with Ye. To be deleted ...
+ *
+ * I need to probably do e2e implementation like this for CCA:
+ *
+ * Option 1: (John prefers this option)
```

Comment:
I think we need to schedule a meeting to discuss about what option to go with, after the CC summit 2023.
We see, we can support both options while there are some additional works needed to realize Option-1.

From our perspective, supporting both would be better, as a TEE platform. Option-2 nicely fits in our entire code base and some internal test cases, while supporting Option-1 would make things easier for the likes of the certifier framework.

After doing some additional work on our side for Option-1, you might be able to access and verify an attestation report as this PR does for verifying the report of the keystone.

Reply:
@jinbpark -- I really do not know how to handled / resolve this feedback, discussion.

I just took notes based on feedback that @yelvmw gave during my initial investigations.

From point of this review, I will just leave this design comment in the code, and we / someone can benefit from it in future. For now, no changes are being done here.

Reply:
> For now, no changes are being done here.

Agreed. You don't need to handle this feedback/discussion in this PR.